### ISSUE TITLE
This repository doesn't do anything valuable.

### ISSUE THREAD

**Issue by almereyda** [RichardHitier/GeoAgenda#1]
I would have loved to see the tempospatial model, as we are investigating similar directions with @transformap.

**Comment by RichardHitier** [comment]
Hi Jon
And thank you for your interest.

GeoAgenda is an old idea.

Currently working on a web client for OpenAgenda: @RichardHitier/OaClient, wip.

## For the tempospatial model I had thought of ical format with just 2 fields for geolocation.

Richard

**Comment by almereyda** [comment]
Yes, but this old idea directly connects to many others having a similar, but vague image of it.

Does OpenAgenda also have Open Sources? Didn't find anything, as it also sounds related.

`ical` is definitely a standard vocabulary, while others investigate how time and space interact as data. https://github.com/kgeographer/topotime
